src/test/java/org/minions/devfund/noemiguzman/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,173 @@
+package org.minions.devfund.noemiguzman.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *  Class to test movie database.
+ */
+public class MovieDatabaseTest {
+    private MovieDatabase moviedb1;
+    private MovieDatabase moviedb2;
+    private Movie movie1;
+    /**
+     * Method to setup movie1 and movie2.
+     */
+
+    @Before
+    public void setUpMovieDatabases() {
+        String actorAffleck = "Ben Affleck";
+        String movieGlass = "Glass";
+        moviedb1 = new MovieDatabase();
+        moviedb2 = new MovieDatabase();
+
+        movie1 = new Movie(movieGlass);
+
+        List<Movie> movieList1 = new ArrayList<>();
+        movieList1.add(movie1);
+
+        Actor actor1 = new Actor(actorAffleck, movieList1);
+        movie1.addActor(actor1);
+
+    }
+
+    /**
+     *  Method to test constructor movie database.
+     */
+    @Test
+    public void testConstructor() {
+        List<Movie> movieList = new ArrayList<>();
+        List<Actor> actorList = new ArrayList<>();
+        assertEquals(movieList, moviedb1.getMovieList());
+        assertEquals(actorList, moviedb2.getActorList());
+    }
+
+    /**
+     *  Method to test add movie.
+     */
+    @Test
+    public void testAddMovieTwoTime() {
+        String actorAffleck = "Ben Affleck";
+        String actorWest = "Adam West";
+        String movieGlass = "Glass";
+        List<Movie> movieList = new ArrayList<>();
+        MovieDatabase moviedb3 = new MovieDatabase();
+        String[] actors1 = new String[] {actorAffleck, actorWest};
+        moviedb3.addMovie(movieGlass, actors1);
+        moviedb3.addMovie(movieGlass, actors1);
+        movieList.add(movie1);
+        List<Movie> actualMovies = moviedb3.getMovieList();
+        assertEquals(movieList.toString(), actualMovies.toString());
+    }
+    /**
+     *  Method to test add movie.
+     */
+    @Test
+    public void testAddMovie() {
+        String actorWest = "Adam West";
+        String actorAffleck = "Ben Affleck";
+        String movieGlass = "Glass";
+        List<Movie> movieList = new ArrayList<>();
+        MovieDatabase moviedb3 = new MovieDatabase();
+        String[] actors1 = new String[] {actorAffleck, actorWest};
+        moviedb3.addMovie(movieGlass, actors1);
+        movieList.add(movie1);
+        List<Movie> actualMovies = moviedb3.getMovieList();
+        assertEquals(movieList.toString(), actualMovies.toString());
+    }
+
+    /**
+     *  Method to test add rating.
+     */
+    @Test
+    public void testAddRating() {
+        String actorWest = "Adam West";
+        String actorAffleck = "Ben Affleck";
+        String movieBatman = "Batman";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Batman" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)